CultivateMVP/JournalOptionsScene/JournalOptionsViewModel.swift

```diff
@@ -1,17 +1,52 @@
 import Foundation
 
 protocol JournalOptionsViewModelType {
-    var entries: [Entry] { get }
-    var moodEntry: MoodEntry { get }
+    // to do - turn into plain UI things
+    var entries: [Entry]? { get }
+    var moodEntry: MoodEntry? { get }
 }
 
-struct JournalOptionsViewModel: JournalOptionsViewModelType {
+class JournalOptionsViewModel: JournalOptionsViewModelType {
 
-    var entries: [Entry]
-    var moodEntry: MoodEntry
+    var entries: [Entry]?
+    var moodEntry: MoodEntry?
+    private var provider: JournalOptionsProvider
```

Comment:
Untestable code because no interface is injected. Try to write a couple of tests for it.